www/artists/jmitchell2/album.html

```diff
@@ -11,36 +11,36 @@
     </head>
     <body>
         <h1>
-            Joni's Album Art &#9829;
+            Joni's Album Art &hearts;
         </h1>
-        <div id="nav">
-            <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="/">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
+        <div id="nav" class="nav">
```

Comment:
Why does this navigation div have a nav class, and none of the others? Particularly when the nav class isn't defined in style.css, and has no effect on anything. (The same can be said about the nav id, but at least it's consistent.)